apps/dashboard/src/lib/utils/redirect.js

```diff
@@ -0,0 +1,11 @@
+import { goto } from '$app/navigation';
+import { globalStore } from './store/app';
+import { get } from 'svelte/store';
+
+let isOrg = get(globalStore);
+
+export function redirectToLesson(settings, courseId) {
+  if (!settings && isOrg.isOrgSite) {
+    goto(`${courseId}/lessons?next=true`);
+  }
+}
```

Comment:
A couple of things here:

1. This code will not always have the latest of value of `isOrg`. Move the state `get` into the function.
```
export function redirectToLesson(settings, courseId) {
  const isOrg = get(globalStore);

  if (!settings && isOrg.isOrgSite) {
    goto(`${courseId}/lessons?next=true`);
  }
}
```
2. Change this file to `.ts` and make each argument have proper type.

3. The first argument `settings` doesn't match its purpose. From the code I can see it is a boolean and it is related whether a customization is enabled or not. I'd suggest: `isPageEnabled`.